src/main/java/com/example/bank/account/Account.java

```diff
@@ -0,0 +1,49 @@
+package com.example.bank.account;
+
+import com.example.bank.registration.jpa.Client;
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+@Entity
+/**  W columnNames podajemy nazwę pola, czy zmapowanej kolumny w bazie danych? */
+@Table(uniqueConstraints = @UniqueConstraint(columnNames = "account_number"))
+@Getter
+@Setter
+@NoArgsConstructor
+public class Account {
+    @Id
+    @SequenceGenerator(name = "account_sequence", sequenceName = "account_sequence", allocationSize = 10)
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "account_sequence")
+    private Long id;
+    /**  Czy id może być typu String i co z inkrementacją tego pola?? Jaki to będzie GenerationType?  */
```

Comment:
Moze byc. 
@Entity
public class MyEntity {
    @Id
    @GeneratedValue(generator = "myGenerator")
    @GenericGenerator(name = "myGenerator", strategy = "com.example.MyGenerator")
    private String id;
    
I klasa MyGenerator musi pelnic funkcje generatora. 
`public class CustomStringIdGenerator implements IdentifierGenerator {

    private static final AtomicLong counter = new AtomicLong(1000); // Starting from 1000 for example
    private static final String PREFIX = "ID-";
    private static final SimpleDateFormat dateFormat = new SimpleDateFormat("yyyyMMddHHmmss");

    @Override
    public Serializable generate(SharedSessionContractImplementor session, Object object) {
        long count = counter.getAndIncrement();
        String timestamp = dateFormat.format(new Date());
        return PREFIX + timestamp + "-" + count;
    }
}`

Powyzej przyklad takiej klasy.
Zadaj sobie inne pytanie, po cholere robic to Stringiem ? :D Czy pytasz po prostu z ciekawosci a nie miales takiego zamyslu ?